livekit-agents/livekit/agents/ipc/proc_pool.py

```diff
@@ -48,6 +48,16 @@ def __init__(
     def processes(self) -> list[SupervisedProc]:
         return self._processes
 
+    def get(self, job_id: str) -> SupervisedProc | None:
+        return next(
```

Comment:
Sorry, by a map I mean to avoid iterating, and creating a dict instead

Reply:
i don't think it makes sense for us here, the proc is created and referred to before a job is assigned to it